maestro/create-account-or-login.yaml

```diff
@@ -0,0 +1,11 @@
+appId: im.status.ethereum.debug
+---
+- runFlow: # to run onboarding flow only when device is in onboarding state
+      when:
+        visible: "I’m new to Status"
+      file: onboarding.yaml
+
+- runFlow: # to run login flow only when device is on login screen
+      when:
```

Comment:
this `when` clause is known to slow things down.
Maybe it makes sense to keep this here and introduce separate `make automate-login` and `make automate-onboarding` 🤔

Reply:
I will work on this in follow ups